rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -8,11 +8,11 @@
             classifications added to them.
         </description>
         <dependencies>
-            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
-            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
         </dependencies>
         <sourceTechnology id="websphere" />
-        <targetTechnology id="eap" versionRange="[6,7)" />
+        <targetTechnology id="eap" versionRange="[6,7]" />
```

Comment:
@jsight Links are going to be updated I create jira issues for this and asked  @sgilda to do that and she did the updates AFAIK for some and others will be filed or updated if needed.

I wasn't sure with the updating to 6 - 8 version range, but I think you both are right, so I will update them to that version range.